jenkins/scripts/integration_test.sh

```diff
@@ -43,7 +43,7 @@ VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-if [ "${TESTS_FOR}" == "feature_tests" ]
+if [ "${TESTS_FOR}" == "feature_tests" ] || [ "${TESTS_FOR}" == "feature_tests_upgrade" ]
```

Comment:
you could catch both by putting a wildcard in the first match

Reply:
done